scripts/deploy.backend.sh

```diff
@@ -14,18 +14,67 @@ case $ENV in
     ;;
 esac
 
+II_CANISTER_ID="$(dfx canister id internet_identity --network "$ENV")"
+POUH_ISSUER_CANISTER_ID="$(dfx canister id pouh_issuer --network "$ENV")"
```

Comment:
ditto

Reply:
Also not, the proper canister id will be set in dfx.json in this case.

Reply:
Right.